From 0e17feb8c08f855fc93c1a006eecebd205e67115 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 23 Jun 2014 17:16:41 +0200
Subject: [PATCH 5/6] Revert "drm: Perform ioctl command validation on the
 stored kernel values"

powervr kernel modules don't define size and direction information for ioctls
(see pvr_drm.h).

So we must use commands as supplied by blobs.

This reverts commit e4fda9f264e154946c678ca7ff07023f573abf6a.
---
 drivers/gpu/drm/drm_drv.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/drm_drv.c b/drivers/gpu/drm/drm_drv.c
index 345be03..303406a 100644
--- a/drivers/gpu/drm/drm_drv.c
+++ b/drivers/gpu/drm/drm_drv.c
@@ -326,7 +326,6 @@ long drm_ioctl(struct file *filp,
 		usize = asize = _IOC_SIZE(cmd);
 		if (drv_size > asize)
 			asize = drv_size;
-		cmd = ioctl->cmd_drv;
 	}
 	else if ((nr >= DRM_COMMAND_END) || (nr < DRM_COMMAND_BASE)) {
 		u32 drv_size;
-- 
1.8.3.1

